beacon-chain/core/blocks/attester_slashing_test.go

```diff
@@ -395,3 +395,149 @@ func TestProcessAttesterSlashings_AppliesCorrectStatusCapella(t *testing.T) {
 	require.Equal(t, uint64(31000000000), newState.Balances()[1])
 	require.Equal(t, uint64(32000000000), newState.Balances()[2])
 }
+
+func TestProcessAttesterSlashings_AppliesCorrectStatusDeneb(t *testing.T) {
```

Comment:
There is a lot of copy pasted code with equivalent functions from other forks.
Could this be factorized?